### PR TITLE
[ch396] Django 2 and django-tenants

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

**Comment by robertavram** [comment]
Please resolve conflicts.

File: src/requirements/base.txt [hunk 1]
```diff
@@ -31,16 +32,16 @@ django-import-export==1.0
...
-django-model-utils==2.6
+django-model-utils==3.1.2
 django-mptt==0.9
 django-ordered-model==1.5
 django-post-office==3.0.4  # via unicef-notification
 django-redis-cache==1.7.1
 django-rest-swagger==2.2
 django-storages==1.6.6
-django-tenant-schemas==1.9.0
+django-tenant-schemas==1.9.0  # via tenant-schemas-celery
```
Comment (reinbach) [inline src/requirements/base.txt]: Looks like the old package `django-tenant-schemas` is being required by another package, maybe we need to look at `tenant-schemas-celery` or find another one so we don't end up with both the old and the new package?

File: src/requirements/input/base.in [hunk 2]
```diff
@@ -49,3 +49,4 @@ unicef_restlib==0.3.0
 unicef_snapshot==0.2.1
 
 -e git+https://github.com/unicef/djangosaml2.git@v0.16.11.2#egg=djangosaml2
+-e git+https://github.com/tomturner/django-tenants.git@master#egg=django-tenants
```
Comment (reinbach) [inline src/requirements/input/base.in]: Could we rather pin this to a specific commit, if we are waiting for a new release

File: src/requirements/base.txt (hunk 1 above)
Comment (domdinicola) [reply src/requirements/base.txt]: yeah... you're right.. what do you suggest?
https://github.com/maciej-gol/tenant-schemas-celery
actually the code here is not too much...

File: src/requirements/input/base.in (hunk 2 above)
Comment (domdinicola) [reply src/requirements/input/base.in]: my idea was temporary, I would prefer to wait a new tag/release

File: src/requirements/base.txt (hunk 1 above)
Comment (reinbach) [reply src/requirements/base.txt]: Maybe implement/duplicate this for `django-tenant`?

**Comment by domdinicola** [comment]
@reinbach with latest version of tenant-celery is not required anymore

**Comment by robertavram** [comment]
Let's have a session dedicated on this. I'll set up a meeting.

**Comment by domdinicola** [comment]
I didn't want to jump too much.. I was thinking to go to 2.1 next/jan sprint

**Pull request merged**